### ISSUE TITLE
chief_creative_director agent in marketing_strategy example is defined in agents.yaml but is not used in crew.py

### ISSUE THREAD

**Issue by matgren** [crewAIInc/crewAI-examples#203]
Thank you for awesome work! while working on marketing_strategy example I've noticed that chief_creative_director agent is defined but never used. Am I missing something or this agent is not really taking part in the process?

**Comment by matgren** [comment]
not sure who supports the community but the issue still exists.

**Comment by injilashah** [comment]
@matgren  Yes there are missing agents and corresponding tasks in __crew.py file.
 just create a new agent for the chief_creative_director agent and a new task  for the same at the end of the predefined agents and predefined tools in ___crew.py file
